### PR TITLE
Simple Hearing Aid

### LINKED ISSUE #74: Parsing of the model JSON File
Sometimes the parsing of the model JSON Files fails. This can be fixed by updating line 97 of `autogen\configureModel.m` to use cell array indexing.

### PR DESCRIPTION
**Pull request by dack-fe**
Created the Simple Hearing Aid Model for the Audio Mini based on the Beamer
 - Implements Noise Suppression on both channels
 - FFT Filtering for the Left Channel 
 - FFT Filtering for the Right Channel
Added Simple Hearing Aid Model Read Me
Added Models Read Me

Working towards Issues #60, #78, & #79

### CODE REVIEW

File: models/simple_hearing_aid/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# A Simple Hearing Aid
+This simple hearing aid example is a combination of the [Adaptive Wiener Filter Noise Suppression Model](../noise_suppression) and the [FFT Filter Model](../fft_filters).
+
```
Comment (fe-wickham) [inline models/simple_hearing_aid/readme.md:4]: Add `This model was developed using the Frost Autogen Framework and deployed to an Audio Mini using Frost Edge. To set up Frost Autogen or Frost Edge, please review the [Getting Started Guides](https://github.com/fpga-open-speech-tools/docs/tree/master/getting_started). `

**Review by fe-tdavis — APPROVED** [review]
Looks good.  I made a couple minor changes to the documentation

**Review by fe-wickham — CHANGES_REQUESTED** [review]
With the register parsing issue fixed, the only thing left is a quick update to the readme

File: models/simple_hearing_aid/readme.md (hunk 1 above)
Comment (fe-wickham) [reply models/simple_hearing_aid/readme.md:4]: Done with 3ec81dd

**Review by fe-wickham — APPROVED** [review]
I added the Frost boilerplate to the ReadMe so everything looks good to me

**Pull request merged**